request.go

```diff
@@ -446,7 +460,8 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 			}
 
 			// As a final catch-all, ensure types line up to avoid a runtime panic.
-			if fieldValue.Kind() != v.Kind() {
+			// Ignore interfaces since interfaces are poly
```

Comment:
polymorphic.  But if you feel that isn't the correct term, a more accurate comment is welcome.  

Why does this matter?
Basically, this "final catch-all" was introduced when I rebased, and it adversely affected our use of use `interface{}` struct fields as a generic JSON object.

We were doing something like this: 
```
type Model struct {
   Raw  interface{}  `jsonapi:"attr,raw"`
}

...	

json.Unmarshal(jsonByteData, model.Raw)

```